src/main/java/inventorysetups/InventorySetupsConfig.java

```diff
@@ -365,11 +365,23 @@ default InventorySetupsShowWornItemsFilterID showWornItemsFilter()
 		return InventorySetupsShowWornItemsFilterID.All;
 	}
 
+	@ConfigItem(
+			keyName = CONFIG_KEY_SECTION_MODE,
+			name = "Submenu in Section mode",
+			description = "Enable submenus when selection mode is enabled",
```

Comment:
Description is not clear. Mention the worn items menu

"Enable section submenus on the worn items button when section mode is enabled"